### PR TITLE
Fix:lack PackageRequest_TwoWay in Write

### PR DESCRIPTION
**Pull request by fangyincheng**
PackageRequest_TwoWay is necessary in hessian.go#Write()

### CODE REVIEW

File: request.go [hunk 1]
```diff
@@ -143,7 +143,11 @@ func packRequest(service Service, header DubboHeader, params interface{}) ([]byt
 	if hb {
```
Comment (wongoo) [inline request.go]: a switch statement would be better

**Comment by wongoo** [comment]
the following Chinese comment can be also change to English:
```
map.go:259:		
map.go:262:		
object.go:131:	
object.go:346:	
object.go:364:	
pojo.go:103:
ref.go:65:
request.go:40:	
request.go:51:	
request.go:75:	
request.go:87:	
request.go:149:	
string.go:60:
```

**Comment by fangyincheng** [comment]
> request.go 文件中看到有 “java 中标识一个class的ID” 之类的中文注释，能否一起改成e文？

ok，I will do it!

File: request.go (hunk 1 above)
Comment (fangyincheng) [reply request.go]: Good. I'll revise it.

**Pull request merged**